sig-multicluster/management-cluster-position-statement.md

```diff
@@ -0,0 +1,46 @@
+# Management Cluster - SIG Multicluster Position Statement
+
+Author: Corentin Debains (**[@corentone](https://github.com/corentone)**), Google  
+Last Edit: 2024/12/09  
+Status: DRAFT
+
+## Goal
+To establish a standard definition for a central cluster that is leveraged by multicluster 
+controllers to manage multicluster applications or features across an inventory of clusters.
+
+## Context
+Multicluster controllers have always needed a place to run. This may happen in external
+proprietary control-planes but for more generic platforms, it has been natural for the
+Kubernetes community to leverage a Kubernetes Cluster and the existing api-machinery
+available. There has been a variety of examples of which we can quote ArgoCD, MultiKueue
+or any of the Federation effort (Karmada, KubeAdmiral), all of them not-naming the "location"
+where they run or not aligning on the name (Admin cluster, Hub Cluster, Manager Cluster...). 
+The [ClusterInventory](https://github.com/kubernetes/enhancements/blob/master/keps/sig-multicluster/4322-cluster-inventory/README.md)
+(ClusterProfile CRDs) is also the starting point for a lot of multicluster controllers and, 
+being a CRD, it requires an api-machinery to host it.
+
+## Definition
+
+A (multicluster) management cluster is a Kubernetes cluster that acts as a 
+control-plane for other Kubernetes clusters (named Workload Clusters to differentiate
+them). It MUST have visibility over the available clusters and MAY have administrative
+privileges over them. It SHOULD not be part of workload clusters to provide a better
+security isolation, especially when it has any administrative privileges over them.
+There MAY be multiple management clusters overseeing the same set of Workload Clusters
```

Comment:
This statement supposes multiplicity only in the form of potentially competing equals; it omits the possibility of clusters fulfilling distinct roles.

Reply:
I tried to clarify in the next section that it could be multiple roles:
> to allow for separation of functionality (security-enforcer management cluster vs
 config-delivery management cluster)
 
 is the wording not strong enough? I don't mean to close that door and thought the current wording was enough and not emitting a direction, just requiring that the admin oversees potential overlap between different management clusters. If there is no overlap, they are fine to co-exist as separate clusters.